spec/requests/topics_controller_spec.rb

```diff
@@ -4841,4 +4841,45 @@ def invite_group(topic, expected_status)
       expect(body["group_name"]).to eq(group.name)
     end
   end
+
+  describe "#set_notifications" do
+    describe "initiated by admin" do
+      it "allows fetches user from params if the request is_api? " do
```

Comment:
```suggestion
      it "admins can update another user's notification level via API" do
```

Reply:
Can we also add a test case where the admin, via API, can update their own notification level? I.e. admin makes API request, without params, and their topic notification level gets updated.

Reply:
I've added this case.